src/main/scala/org/mdedetrich/apache/sonatype/ApacheSonatypePlugin.scala

```diff
@@ -134,13 +134,6 @@ object ApacheSonatypePlugin extends AutoPlugin {
             .toList
         }
       },
-      // See https://issues.apache.org/jira/browse/LEGAL-28
-      packageSrc / mappings ++= {
-        Seq(
-          apacheSonatypeLicenseFile.value -> "META-INF/LICENSE",
-          apacheSonatypeNoticeFile.value  -> "META-INF/NOTICE"
-        ) ++ apacheSonatypeDisclaimerFile.value.map(path => path -> "META-INF/DISCLAIMER").toSeq
-      },
       packageDoc / mappings ++= {
```

Comment:
@raboof any idea if packageDoc could be affected too (next lines)?

Reply:
It doesn't seem like it is: when removing these mappings the scripted test fails - which I guess makes some sense as the `packageSrc` jar contains "sources and resources" while the `packageDoc` jar contains "API documentation".